docs/user/integration/dynatrace/README.md

```diff
@@ -101,6 +101,8 @@ kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="a
 
 ### Ingest Traces
 
+To start ingesting custom pans and Istio spans, you need to enable the Istio tracing feature and then deploy a TracePipeline to enable the shipment to Dynatrace using the secrets defined above.
```

Comment:
This sentence seems to be a summary of the steps that happen in this section? Then I'd shorten it a bit and add the information about the Secret closer to line 128 where it happens.

```suggestion
To start ingesting custom spans and Istio spans, you must enable the Istio tracing feature and then deploy a TracePipeline.
```